src/wasm_runner.rs

```diff
@@ -64,10 +64,10 @@ pub fn prepare_stdio_streams(
             .append(true)
             .create(true)
             .open(log_dir.join(STDERR_FILE))?,
-        ambient_authority(),
```

Comment:
Is this little guy a wasmtime API change or did he just sneak in?

Reply:
Yeah, it's the way to express capabilities in `cap-std` (used by Wasmtime).